lib/edifact_rails/serializer.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module EdifactRails
+  class Serializer
+    # Treat the input a little, split the input string into segments, parse them
+    def serialize(segments, with_service:)
+      # Add the UNA segment
```

Comment:
move this comment down to line 13, perhaps quickly explain purpose of with_service parameter here